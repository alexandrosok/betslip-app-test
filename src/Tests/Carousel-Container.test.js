import React from 'react';
import {shallow, configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16'
import {shallowToJson} from 'enzyme-to-json';
import CarouselContainer from "../components/Carousel-Container/carousel-container";

configure({adapter: new Adapter()})


describe('CarouselContainer', () => {
    const component = shallow(
        <CarouselContainer/>
    );

    it('should render correctly', () => {
        expect(shallowToJson(component)).toMatchSnapshot();
    });

    it('state should contain', () => {
        expect(component.state()).toEqual({events: []});
    });
});
