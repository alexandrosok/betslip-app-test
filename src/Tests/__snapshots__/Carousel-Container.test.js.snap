// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CarouselContainer should render correctly 1`] = `
<div
  className="carousel-container"
>
  <Gallery />
</div>
`;
